java/clients/src/main/java/sleeper/clients/teardown/TerminateEMRServerlessApplications.java

```diff
@@ -104,6 +114,16 @@ private boolean allApplicationsTerminated() {
         return applicationsStillRunning == 0;
```

Comment:
It looks like this will always be true, because listActiveApplications only returns applications with a state of STARTED, but this filters out applications that aren't STOPPED. That means applicationsStillRunning will always be 0.

Reply:
I believe I have fixed this